day1/es1/LocalButton.qml

```diff
@@ -0,0 +1,33 @@
+import QtQuick 2.0
+
+Rectangle {
+    id: button
+
+    property string stateName: "unchecked"
+    property color colorForTheStateChecked: "lightgray"
+    property color colorForTheStateUnchecked: "lightgray"
+    property string text: qsTr("none")
+
+    color: switchButton.state == "checked"? colorForTheStateChecked: colorForTheStateUnchecked
```

Comment:
Anche qui stiamo usando `switchButton` come un ID globale, dome dicevo poco più in basso.
Come dicevo per l'altra casistica, questo possiamo gestirlo fuori da questa componente.
Riesci a gestire questa soluzione da solo, dopo aver visto l'altra? :)

Reply:
spero di si, se ho problemi ti faccio sapere

Reply:
Vai, perfetto

Reply:
eliminato utlizzo ID globale `color: switchButton.state == "checked"? "blue": "lightgray"`